From 35c28be32762d5af6b93cb6a3420c5977f7bf599 Mon Sep 17 00:00:00 2001
From: Rafal Luzynski <digitalfreak@lingonborough.com>
Date: Mon, 13 May 2019 23:11:28 +0200
Subject: [PATCH] tests: Update month name check for Greek locale
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Update the abbreviated month name in the test to match the actual
translation. Otherwise the test fails with false positive.

Vocabulary:

July (nominative)  - Greek: Ιούλιος (abbreviated: Ιούλ)
Of July (genitive) - Greek: Ιουλίου (abbreviated: Ιουλ)

This is similar to commit 4d215e006ee69bd7a993bf853722993433adbf9a
and commit 7fe793e125c316ac34edd8158df5a132cb044bc1.

Closes #1776
---
 glib/tests/date.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glib/tests/date.c b/glib/tests/date.c
index 5db9d4c..cb1efc9 100644
--- a/glib/tests/date.c
+++ b/glib/tests/date.c
@@ -387,7 +387,7 @@ test_month_names (void)
       TEST_DATE ( 1,  4, 2018,   "%OB %Y",       "Απρίλιος 2018");
       TEST_DATE ( 1,  5, 2018,   "%OB %Y",          "Μάιος 2018");
       TEST_DATE ( 1,  6, 2018,   "%OB %Y",        "Ιούνιος 2018");
-      TEST_DATE (16,  7, 2018, "%e %b %Y",        "16 Ιούλ 2018");
+      TEST_DATE (16,  7, 2018, "%e %b %Y",        "16 Ιουλ 2018");
       TEST_DATE ( 1,  8, 2018,   "%Ob %Y",            "Αύγ 2018");
     }
   else
-- 
2.25.1

